[buildout]
prefer-final = false
newest = false
download-cache = .cache
develop = .
parts = 

[project]
name = infi.exceptools
homepage = https://github.com/Infinidat/${project:name}
company = Infinidat
namespace_packages = ['infi']
install_requires = ['infi.pyutils',
	'mock',
	'setuptools']
version_file = src/infi/exceptools/__version__.py
description = a mechanism for chaining exceptions, compatible with Python 2.x
long_description = Python 3.x has such mechanism, which was not ported to Python 2.x, this is an alternative
console_scripts = []
gui_scripts = []
package_data = []
upgrade_code = None
product_name = ${project:name}
post_install_script_name = None
pre_uninstall_script_name = None

[isolated-python]
recipe = infi.recipe.python
version = v3.5.1.10

[setup.py]
recipe = infi.recipe.template.version
input = setup.in
output = setup.py

[__version__.py]
recipe = infi.recipe.template.version
output = ${project:version_file}

[development-scripts]
dependent-scripts = true
recipe = infi.recipe.console_scripts
eggs = infi.traceback
	${project:name}
	coverage
	ipython
	pylint
	nose
interpreter = python

[pack]
recipe = infi.recipe.application_packager

[sublime]
recipe = corneti.recipes.codeintel
eggs = ${development-scripts:eggs}

